src/backend/marsha/bbb/serializers.py

```diff
@@ -224,6 +225,28 @@ class Meta:  # noqa
         read_only=True, pk_field=serializers.CharField()
     )
 
+    @property
+    def max_upload_file_size(self):
```

Comment:
There is no reason to use this property IMO. You can directly use `settings.CLASSROOM_DOCUMENT_SOURCE_MAX_SIZE` line 240